test/Utility/TypeExtensionsTests.cs

```diff
@@ -419,6 +508,92 @@ public void TestObjectToTypedDataJsonHashtable()
 
             Assert.Equal(expected, input.ToTypedData(manager));
         }
+
+        [Fact]
+        public void TestObjectToTypedDataJsonPSObject1()
```

Comment:
Maybe be a little more discriptive here and the others?

`TestHashtablePSObjectToTypedDataJson`

Reply:
Sure, will change them.